contrib/guix/manifest.scm

```diff
@@ -2,7 +2,7 @@
              ((gnu packages bash) #:select (bash-minimal))
              (gnu packages bison)
              ((gnu packages certs) #:select (nss-certs))
-             ((gnu packages cmake) #:select (cmake-minimal))
+             ((gnu packages cmake) #:select (cmake-3.30))
```

Comment:
764af91cfe5dcde82a9ff9fe810a9ad1d579b3ac

Do we really want to switch from `cmake-minimal`, which brings in additional inputs such as `ncurses`, `python-sphinx` and `texinfo`? I've really appreciated the current setup since https://github.com/bitcoin/bitcoin/commit/2c9eb4afe1f583aafa552b2711b149f17ef8320f.

FWIW, `cmake-minimal` was updated to 3.31.8 in https://codeberg.org/guix/guix/commit/3441c3e30d24c82c159d5d0049692dfd6bacd31c.